libqtile/window.py

```diff
@@ -465,12 +465,11 @@ def place(self, x, y, width, height, borderwidth, bordercolor,
             width=width,
             height=height,
         )
-        if above:
-            kwarg['stackmode'] = StackMode.Above
-
         self.window.configure(**kwarg)
         self.paint_borders(bordercolor, borderwidth)
 
+        if above:
+            self.qtile.change_layer(self.window.wid)
```

Comment:
We're already doing two ConfigureWindow in two lines, let's not add a third one. :)

Reply:
I don't know what happens in the backend - is this worth thinking about?

In general I agree, but the only solution I see at the moment would be that `change_layer` just helps gather parameters for the eventual call to `configure` instead of actively changing the layer. Do you have something better in mind?

Reply:
The current implementation avoids the third call. Please have a look and let me know whether you really prefer it - I'm not decided on it yet.^^